Formula/dvm.rb

```diff
@@ -3,6 +3,7 @@ class Dvm < Formula
   homepage "https://github.com/getcarina/dvm"
   url "https://github.com/getcarina/dvm/archive/0.6.4.tar.gz"
   sha256 "7ff3bc5271432b7f5da535cd04228bf212dd2965ed15998837d660452094ff0e"
+  revision 1
```

Comment:
Sure, I dropped the revision. Need to push a rebottle then, since the content has changed.